test/unit/search/search_test.cpp

```diff
@@ -278,16 +295,75 @@ TYPED_TEST(search_test, search_strategy_best)
 {
     using hits_result_t = std::vector<typename TypeParam::size_type>;
 
+    hits_result_t possible_hits{0, 4, 8}; // any of 0, 4, 8 ... 1, 5, 9 are not best hits
     {
         configuration const cfg = max_error{total{1}} | mode{best};
 
-        hits_result_t possible_hits{0, 4, 8}; // any of 0, 4, 8 ... 1, 5, 9 are not best hits
         hits_result_t result = search("ACGT"_dna4, this->index, cfg);
         ASSERT_EQ(result.size(), 1u);
         EXPECT_TRUE(std::find(possible_hits.begin(), possible_hits.end(), result[0]) != possible_hits.end());
 
         EXPECT_EQ(search("AAAA"_dna4, this->index, cfg), (hits_result_t{})); // no hit
     }
+    // Find best match with 1 insertion at the end.
+    {
```

Comment:
can you please move the comment on the same line as the opening braces and leave the empty line between the scopes. For all the following scopes as well.